nuvlaedge/agent/orchestrator/kubernetes.py

```diff
@@ -37,23 +37,48 @@ class KubernetesClient(COEClient):
 
     WAIT_SLEEP_SEC = 2
 
-    # FIXME: This needs to be parametrised.
+    # FIXME: This needs to be parameterised.
     NE_DB_ROOT_HOSTPATH = '/var/lib/nuvlaedge'
     NE_DB_CONTAINER_PATH = str(FILE_NAMES.root_fs)
 
+    DEFAULT_IMAGE_PULL_POLICY = "IfNotPresent"
```

Comment:
Should we not put "Always" here as well ?

Reply:
Ah rats... Yes, you're right. I've changed it and pushed. 
I suppose having "Always" makes me a bit nervous...